project/build.properties

```diff
@@ -16,4 +16,4 @@
 # limitations under the License.
 #
 
-sbt.version=0.13.5
+sbt.version=0.13.7
```

Comment:
This is the minimum version that scalajs projects were using, i ran into problems with 0.13.5.